resalloc_ibm_cloud/ibm_cloud_vm.py

```diff
@@ -339,6 +339,11 @@ def main():
         opts.instance_name = name
         opts.instance = "production" if "-prod-" in name else "devel"
 
+    maxlen = 56
```

Comment:
is it really 56? It says 63 in UI
![Screenshot_20231024_203853](https://github.com/fedora-copr/resalloc-ibm-cloud/assets/70757578/5b6c01bd-916b-4550-ad3d-65433ce3b2dc)